feathr_project/feathr/utils/job_utils.py

```diff
@@ -7,28 +7,35 @@
 import tempfile
 
 
-def get_result_df(client: FeathrClient, format: str = None, res_url: str = None) -> pd.DataFrame:
+def get_result_df(client: FeathrClient, format: str = None, res_url: str = None, local_folder: str = None) -> pd.DataFrame:
     """Download the job result dataset from cloud as a Pandas dataframe.
 
     format: format override, could be "parquet", "delta", etc.
     res_url: output URL to download files. Note that this will not block the job so you need to make sure the job is finished and result URL contains actual data.
+    local_folder: optional parameter to specify the absolute download path. if the user does not provide this, function will create a temporary directory and delete it after reading the dataframe.
```

Comment:
nit: can it also be relative? If both, will give it an example better to explain this?

Reply:
@hangfei I thought about whether to use relative path or absolute path, and feel absolute path should able to cover most of the use cases (actually @shivamsanju 's original implementation is for relative path and I asked him to change to absolute path).